### ISSUE TITLE
Problem with DirichletMultinomial package

### ISSUE THREAD

**Issue by andigoni** [andigoni/MeinteR#3]
make: [dirichlet_fit_main.o] Error 1 ERROR: compilation failed for package ‘DirichletMultinomial’

**Comment by andigoni** [comment]
Install package manually

gsl-config: command not found 
Install gsl-devel: yum install gsl-devel(CentOS 7)

Install libgsl-dev: sudo apt-get install libgsl-dev (Ubuntu Desktop)

**Comment by panpipe1997** [comment]
Hi, I recently came across the same problem. 
`apt-get install gsl-bin libgsl-dev` work for me, though it has been 1 year since you postes this issue.

**Comment by shichuanping** [comment]
> Hi, I recently came across the same problem. `apt-get install gsl-bin libgsl-dev` work for me, though it has been 1 year since you postes this issue.
you are right. I also meet the same problem and I use your way to solve it successfully.

**Comment by andigoni** [comment]
Great, thank you!

**Comment by drelo** [comment]
just for new people with this issue this still works fixing the issue on Fedora
`sudo dnf install gsl-devel`
